src/Examples/Warden.Examples.Console/Program.cs

```diff
@@ -60,6 +62,7 @@ private static IWarden ConfigureWarden()
                         .OnCompletedAsync(check => WebsiteHookOnCompletedAsync(check))
                         .OnFailureAsync(check => WebsiteHookOnFailureAsync(check));
                 })
+                .AddPortAvailabilityWatcher("www.google.pl", bld => bld.ForPort(80))
```

Comment:
Rename the project simply to _Warden.Watchers.Port_. 
Currently, it's _Warden.Watchers.ServerStatus_ and there are _PortAvailability_ classes + extensions like _AddPortAvailabilityWatcher()_ (inconsistent namings). 
Everything should be named like _Port_ e.g. _AddPortWatcher()_ or _PortCheckResult_.